keyboards/ergodox_infinity/keymaps/dudeofawesome/keymap.c

```diff
@@ -2,19 +2,7 @@
 #include "debug.h"
 #include "action_layer.h"
```

Comment:
Neither of these lines should be needed. QMK_KEYBOARD_H includes both of these files.

Reply:
Hmm, interesting. I think that's from the input club keymap that I based mine off of. 
I'll try removing them.

Reply:
works fine with imports removed!